recipes-sota/config/files/10-lite-public-stream.toml.in

```diff
@@ -3,6 +3,7 @@ polling_sec = 3600
 repo_server = "https://api.foundries.io/ota/repo/lmp/api/v1/user_repo/"
 
 [pacman]
-type = "ostree"
+type = "ostree+docker-app"
+docker_apps_root = "/var/sota/docker-apps"
 ostree_server = "https://api.foundries.io/ota/treehub/lmp/api/v2/"
```

Comment:
I think we need docker_apps = "" as well?

Reply:
I have that in my /var/sota/sota.toml file per the documentation

Reply:
we don't have to define docker_apps. This defaults to nothing:
 https://github.com/foundriesio/aktualizr/blob/eb5e8b5bc0106ce45450d894c846f1aecbd6191f/src/libaktualizr/package_manager/packagemanagerconfig.cc#L42